libraries/testing/tester.cpp

```diff
@@ -28,8 +28,9 @@ namespace eosio { namespace testing {
       }
       return res;
    }
-
-   base_tester::base_tester(chain_controller::runtime_limits limits) {
+   
```

Comment:
Can we get rid of some of this extra whitespace and extra newlines that was added? (would make the diff of actual code a lot easier to see)

Reply:
Sorry about that, will clean it up.